src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -50,6 +50,17 @@ trait Backend {
   def initializeForWorkflow(workflow: WorkflowDescriptor): HostInputs
 
   /**
+   * Return the implementation of all WDL functions (e.g. read_lines(), stdout()) used
+   * by this backend
```

Comment:
Despite what I said below I think it probably would be good to explain here why this exists